aip/general/0193.md

```diff
@@ -58,7 +58,11 @@ following:
        {
         "@type": "type.googleapis.com/google.rpc.LocalizedMessage",
         "locale": "en-US",
-        "message": "A e2-medium with local-ssd=3,nvidia-t4=2 is currently unavailable in the us-east1-a zone.\nCapacity changes frequently, so try your request in a different zone, with a different VM hardware configuration, or at a later time. For more information, see troubleshooting documentation."
+        "message": "An <e2-medium> VM instance with <local-ssd=3,nvidia-t4=2> is currently unavailable in the <us-east1-a> zone.\n Consider trying your request in the <us-central1-f, us-central1-c> zone(s), which currently has/have capacity to accommodate your request.\n Alternatively, you can try your request again with a different  VM hardware configuration or at a later time. For more information, see the troubleshooting documentation.",
```

Comment:
I think we'd need metadata of "zonesWithCapacity" with a value of "us-central1-f,us-central1-c". I've deliberately removed the space there, and we'd need to do so in the message too - because that metadata should be as easy to handle in software as possible.

Reply:
done, thank you!